Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -32,8 +33,57 @@ using namespace o2::aod;
 using namespace o2::analysis;
 using namespace o2::analysis::hf_cuts_single_track;
 
+/// Event selection
+struct HfProduceSelCollisions {
```

Comment:
(Minor comment) I would call it HfTagSelCollisions, and similarly for the track select task. Just to avoid that one thinks that this creates track tables